src/LaunchDarkly.EventSource/EventSource.cs

```diff
@@ -125,8 +125,8 @@ public EventSource(Configuration configuration)
                 }
                 catch (Exception e)
                 {
-                    _logger.LogError("Encountered an error connecting to EventSource: {0}", e.Message);
-                    _logger.LogDebug(e.ToString());
+                    _logger.LogError( 0, e, "Encountered an error connecting to EventSource: {0}", e.Message);
+                    _logger.LogDebug( "{0}", e);
```

Comment:
As mentioned above, the exception should just be passed into the LogError method, but if you are going to log the raw error details this way, you only want to call ToString() if the Debug log level is enabled.

By switching this to a format string, you're guaranteed